jir_refractive_image_geometry/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!-- <package format="2"> -->
+<package>
+  <name>jir_refractive_image_geometry</name>
+  <version>0.0.0</version>
+  <description>The jir_refractive_image_geometry package</description>
+
+  <maintainer email="robotics@jacobs-university.de">Jacobs Robotics</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>CC BY-NC-ND 4.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>eigen</build_depend>
+  <run_depend>eigen</run_depend>
+  <build_depend>image_geometry</build_depend>
+  <run_depend>image_geometry</run_depend>
+  <build_depend>roscpp</build_depend>
+  <run_depend>roscpp</run_depend>
+  <build_depend>jir_refractive_image_geometry_msgs</build_depend>
+  <run_depend>jir_refractive_image_geometry_msgs</run_depend>
+  <build_depend>libgsl</build_depend>
+
+
+  <!-- The export tag contains other, unspecified, tags -->
+  <export>
+    <!-- You can specify that this package is a metapackage here: -->
+    <!-- <metapackage/> -->
+
+    <!-- Other tools can request additional information be placed here -->
+
+  </export>
+</package>
```

Comment:
no new line at end of file.

is this a third party package?

Reply:
Yes